scripts/tools/annotation/annotation.py

```diff
@@ -8,33 +8,38 @@
   annotation.py (-h | --help)
 
 Options:
-  --data_path=DATA_PATH     Full path to the data-set folder
+  --data_path=DATA_PATH     Complete or relative path to the data-set folder
   --split=SPLIT             Type of split to collect videos from, either `train` or `valid`
   --label=LABEL             Class-label to get the videos for annotation
 """
-from docopt import docopt
-from flask import Flask, request, flash, redirect, url_for
-from flask import render_template, send_from_directory
-import os
-from os.path import join
-import json
+
 import glob
+import json
 import numpy as np
-from sklearn.linear_model import LogisticRegression
-from joblib import dump, load
+import os
 
+from docopt import docopt
+from flask import Flask
+from flask import redirect
+from flask import render_template
+from flask import request
+from flask import send_from_directory
+from flask import url_for
+from joblib import dump
+from joblib import load
+from os.path import join
+from sklearn.linear_model import LogisticRegression
 
 args = docopt(__doc__)
-dataset_path = args['--data_path']
+dataset_path = join(os.getcwd(), args['--data_path'])
```

Comment:
will it work if you give an absolute path ?

Reply:
Yes, I did try that and it worked in both cases:
- absolute path
- relative path

`os.getcwd()` and `os.path.join()` overlaps the parent path and that's why this works.